### DISCUSSION TITLE
Saving to Wrong Folder?

### DISCUSSION THREAD

**Discussion by DekHog-One** [localsend/localsend#2113] [Discussion (English, International)] [answered]
The app works great transferring videos from PC to Android phone, but refuses to use the folder I select for files (Downloads/LocalSend) - every time it saves to Movies folder no matter which folder I choose in app settings?

**Comment by Tienisto** [comment, marked as answer]
You need to disable "Save media to gallery"

**Reply by DekHog-One** [reply]
Perfect, thank you very much.... 👍🏻
